app/server/api/auth_api.py

```diff
@@ -329,6 +330,16 @@ def post(self):
         password_empty = password == '' or password is None
         pin_empty = pin == '' or pin is None
 
+        ratelimit_key = email or post_data.get('phone')
+        if ratelimit_key:
+            limit = rate_limit("login_"+ratelimit_key, 25)
+            if limit:
+                response_object = {
+                    'status': 'fail',
+                    'message': f'Please try again in {limit} minutes'
+                }
+                return make_response(jsonify(response_object)), 403
```

Comment:
Instead of repeating the error message, this would make more sense as a wrapper function that can be applied to the API method